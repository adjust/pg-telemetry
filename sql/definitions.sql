set client_min_messages to warning;
create extension pgtelemetry cascade;
select count(*) > 0 from pgtelemetry.database_size;
SELECT count(*) > 0 from pgtelemetry.relation_total_size;
SELECT count(*) > 0 from pgtelemetry.catalog_total_size;
SELECT count(*) > 0 from pgtelemetry.index_size;
SELECT count(*) > 0 from pgtelemetry.relation_toast_size;
SELECT count(*) > 0 from pgtelemetry.tablespace_size;
SELECT count(*) > 0 from pgtelemetry.database_size;
SELECT count(*) > 0 from pgtelemetry.connections_by_application;
SELECT count(*) > 0 from pgtelemetry.connections_by_state;
SELECT count(*) > 0 from pgtelemetry.connections_by_ip_source;
SELECT count(*) > 0 from pgtelemetry.longest_running_active_queries;
SELECT count(*) > -1 from pgtelemetry.waiting_connections_by_event_type;
SELECT count(*) > 0 from pgtelemetry.locks_by_type;
SELECT count(*) > 0 from pgtelemetry.locks_by_mode;
SELECT pgtelemetry.count_waiting_on_locks_more_than_seconds(300) >= 0;
SELECT count(*) > 0 from pgtelemetry.tuple_access_stats;
SELECT count(*) > 0 from pgtelemetry.autovacuum_stats;
SELECT count(*) > 0 from pgtelemetry.statement_query_rows_time;
SELECT count(*) > 0 from pgtelemetry.statement_query_buffers;
