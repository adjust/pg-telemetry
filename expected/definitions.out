set client_min_messages to warning;
create extension pgtelemetry cascade;
select count(*) > 0 from pgtelemetry.database_size;
 ?column? 
----------
 t
(1 row)

SELECT count(*) > 0 from pgtelemetry.relation_total_size;
 ?column? 
----------
 t
(1 row)

SELECT count(*) > 0 from pgtelemetry.catalog_total_size;
 ?column? 
----------
 t
(1 row)

SELECT count(*) > 0 from pgtelemetry.index_size;
 ?column? 
----------
 t
(1 row)

SELECT count(*) > 0 from pgtelemetry.relation_toast_size;
 ?column? 
----------
 t
(1 row)

SELECT count(*) > 0 from pgtelemetry.tablespace_size;
 ?column? 
----------
 t
(1 row)

SELECT count(*) > 0 from pgtelemetry.database_size;
 ?column? 
----------
 t
(1 row)

SELECT count(*) > 0 from pgtelemetry.connections_by_application;
 ?column? 
----------
 t
(1 row)

SELECT count(*) > 0 from pgtelemetry.connections_by_state;
 ?column? 
----------
 t
(1 row)

SELECT count(*) > 0 from pgtelemetry.connections_by_ip_source;
 ?column? 
----------
 t
(1 row)

SELECT count(*) > 0 from pgtelemetry.longest_running_active_queries;
 ?column? 
----------
 t
(1 row)

SELECT count(*) > -1 from pgtelemetry.waiting_connections_by_event_type;
 ?column? 
----------
 t
(1 row)

SELECT count(*) > 0 from pgtelemetry.locks_by_type;
 ?column? 
----------
 t
(1 row)

SELECT count(*) > 0 from pgtelemetry.locks_by_mode;
 ?column? 
----------
 t
(1 row)

SELECT pgtelemetry.count_waiting_on_locks_more_than_seconds(300) >= 0;
 ?column? 
----------
 t
(1 row)

SELECT count(*) > 0 from pgtelemetry.tuple_access_stats;
 ?column? 
----------
 t
(1 row)

SELECT count(*) > 0 from pgtelemetry.autovacuum_stats;
 ?column? 
----------
 t
(1 row)

SELECT count(*) > 0 from pgtelemetry.statement_query_rows_time;
 ?column? 
----------
 t
(1 row)

SELECT count(*) > 0 from pgtelemetry.statement_query_buffers;
 ?column? 
----------
 t
(1 row)

